react-charts/area-chart/step7.md

```diff
@@ -1,57 +1,14 @@
-Now we'll add a legend to the area chart...
+Change the area chart theme color.
 
-## Task
+1) <strong>Make sure the `App.js` file is still open.</strong>
 
-1) Make sure the App.js file is still open
+2) <strong>Locate the `<Chart>` component and add a property to change the theme.</strong>
 
-2) Locate the code that looks like the following:
-
-<pre class="file">
-&lt;Chart
-  constrainToVisibleArea
-  containerComponent={&lt;ChartVoronoiContainer labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}
-  domainPadding={{ x: [30, 25] }}
-  padding={{
-    bottom: 75,
-    left: 75,
-    right: 50,
-    top: 50
-  }}
-  height={250}
-  width={600}
-&gt;
-</pre>
-
-3) Replace the following property in that section:
-
-<pre class="file" data-target="clipboard">
-padding={{
-  bottom: 75,
-  left: 75,
-  right: 200,
-  top: 50
-}}
-</pre>
-
-- This will add more padding to the right property to accommodate the legend.
-
-4) Add the following properties to that section:
+The `themeColor` property specifies the theme color. Valid values are 'blue,' 'green,' 'multi,' etc. It should look like the following:

```

Comment:
Since we aren't showing all of the valid values, should we reword and also use the enum values instead of strings:
```suggestion
The `themeColor` property specifies the theme color. Some valid values are `ChartThemeColor.blue`,  `ChartThemeColor.green`, or `ChartThemeColor.multi`. It should look like the following:
```